samples/sensor/bme280/boards/intel_rpl_p_crb.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c1 {
+        bme280@76 {
+                compatible = "bosch,bme280";
+                reg = <0x76>;
+		status = "okay";
+        };
```

Comment:
Mixing tabs and spaces. Please use tabs to indent

Reply:
Fixed @bindushree98  commit, also fixed another overlay with similar issue.